Challenge-4/README.md

```diff
@@ -0,0 +1,41 @@
+# Introduction
+
+Welcome to the fourth and final challenge of Summer of Spin 2024!
+
+In this challenge, you will get an intro to Spin Key-Value store by building an application that allows your favorite football (soccer) team to keep track of their players calorie intake during the summer break.
+
+## Spec
+
+It's summer time and your local football league is on break. For players, it's a time to rewind after the long season of intense games. Your local team is worried the players may become unfit by the time they resume for the next season. To ensure that they give the best personalized recovery to all the players when the season resume, they have decided to build an application to keep track of the players' calorie in-take.
+
+Now, it's your job to build this application as a great programmer and supporter of the team!
+
+The application needs to have two endpoints:
+
+- `POST /register {player_name: string, player_no: int}`: This endpoint allows the head coach to register a player by providing their name and number in the request body.
+
+  The response for this endpoint should look like:
+
+  - `200`: `{"success": true, "data": {"player_name": "<PLAYER_NAME>", "player_no": "<PLAYER_NO>"}}`
+  - `400`: `{"success": false}`
```

Comment:
I do not think that the `success` field adds value as the status can be inferred from the response code. there could be a reason filed explaining why it failed. Also `409` might be more appropriate (not too sure)

https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/409

Reply:
Removed the `success` field. I agree that the response code should be 409, updated that as well and will review it if there are any oppositions.